assets/javascripts/discourse/components/user-menu/assigns-list.js

```diff
@@ -4,11 +4,18 @@ import UserMenuNotificationItem from "discourse/lib/user-menu/notification-item"
 import UserMenuAssignItem from "discourse/plugins/discourse-assign/discourse-assign/lib/user-menu/assign-item";
 import Notification from "discourse/models/notification";
 import I18n from "I18n";
-import showModal from "discourse/lib/show-modal";
 import Topic from "discourse/models/topic";
 import UserMenuAssignsListEmptyState from "./assigns-list-empty-state";
+import { withPluginApi } from "discourse/lib/plugin-api";
 
 export default class UserMenuAssignNotificationsList extends UserMenuNotificationsList {
+  constructor() {
+    super(...arguments);
+    withPluginApi("1.6.1", (api) => {
+      api.displayDismissNotificationConfirmation();
+    });
```

Comment:
Was this meant to be placed in the constructor?

Reply:
Yes, it's just an initial value being set. So constructor fits the bill.